mitiq/calibration/settings.py

```diff
@@ -3,13 +3,14 @@
 # This source code is licensed under the GPL license (v3) found in the
 # LICENSE file in the root directory of this source tree.
 
+from copy import deepcopy
 from dataclasses import dataclass, asdict
 from functools import partial
 from typing import Any, Callable, cast, List, Dict
 from enum import Enum, auto
 
 import networkx as nx
-import cirq
+from cirq import Circuit, LineQubit, measure, CNOT, CZ
```

Comment:
I had changed it because I find the `cirq.` ... to be harder to read in a chain of Cirq functions, but I agree with your concern about ambiguity, so I've changed back to importing `cirq` as a top-level object.